### ISSUE TITLE
微调baichuan2-13b,loss一直为0

### ISSUE THREAD

**Issue by yfangZhang** [percent4/llm_open_triplet_extraction#2]
{
    "output_dir": "output/firefly-baichuan2-13b-sft-qlora",
    "model_name_or_path": "/root/models/Baichuan2-13B-Base",
    "train_file": "/root/zhyifang/llm_open_triplet_extraction-main/data/spo.jsonl",
    "template_name": "baichuan2",
    "num_train_epochs": 10,
    "per_device_train_batch_size": 4,
    "gradient_accumulation_steps": 2,
    "learning_rate": 1e-4,
    "max_seq_length": 550,
    "logging_steps": 100,
    "save_steps": 100,
    "save_total_limit": 1,
    "lr_scheduler_type": "constant_with_warmup",
    "warmup_steps": 100,
    "lora_rank": 64,
    "lora_alpha": 16,
    "lora_dropout": 0.05,

    "gradient_checkpointing": true,
    "disable_tqdm": false,
    "optim": "paged_adamw_32bit",
    "seed": 42,
    "fp16": false,
    "bf16": true,
    "report_to": "tensorboard",
    "dataloader_num_workers": 0,
    "save_strategy": "steps",
    "weight_decay": 0,
    "max_grad_norm": 0.3,
    "remove_unused_columns": false
}
![image](https://github.com/percent4/llm_open_triplet_extraction/assets/65322222/d6a240b7-d015-41f6-ae05-cfd3b3df78f0)

**Comment by percent4** [comment]
从你的描述中，我暂时没法找到原因，请你对照下：1. 数据集的格式是否正确；2. 配置文件路径及内容是否正确； 3. 参考下网上公开的资料找找原因。

**Comment by lh5533223** [comment]
> { "output_dir": "output/firefly-baichuan2-13b-sft-qlora", "model_name_or_path": "/root/models/Baichuan2-13B-Base", "train_file": "/root/zhyifang/llm_open_triplet_extraction-main/data/spo.jsonl", "template_name": "baichuan2", "num_train_epochs": 10, "per_device_train_batch_size": 4, "gradient_accumulation_steps": 2, "learning_rate": 1e-4, "max_seq_length": 550, "logging_steps": 100, "save_steps": 100, "save_total_limit": 1, "lr_scheduler_type": "constant_with_warmup", "warmup_steps": 100, "lora_rank": 64, "lora_alpha": 16, "lora_dropout": 0.05,
> 
> ```
> "gradient_checkpointing": true,
> "disable_tqdm": false,
> "optim": "paged_adamw_32bit",
> "seed": 42,
> "fp16": false,
> "bf16": true,
> "report_to": "tensorboard",
> "dataloader_num_workers": 0,
> "save_strategy": "steps",
> "weight_decay": 0,
> "max_grad_norm": 0.3,
> "remove_unused_columns": false
> ```
> 
> } ![image](https://private-user-images.githubusercontent.com/65322222/309653815-d6a240b7-d015-41f6-ae05-cfd3b3df78f0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA1MTcxNTksIm5iZiI6MTcyMDUxNjg1OSwicGF0aCI6Ii82NTMyMjIyMi8zMDk2NTM4MTUtZDZhMjQwYjctZDAxNS00MWY2LWFlMDUtY2ZkM2IzZGY3OGYwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA5VDA5MjA1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgwYmE2ZmQxNzUzZGJkOGU0NWU5YWUyNmQ5NDAxOWYxYTc3YzRjOTcyNGNmNTY0NDlmNjRiNDM5NGQzNTdhMTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.RkzaGh-oSM57gvSdOHDuWhXGPTTI_d0ytSvy8rDwhLg)

请问你是如何训练起来的，是根据Firefly项目中的训练命令吗？是不是还需要安装deepspeed，还需要从hugging Face里边下载相应权重文件吗？
